### PR TITLE
Reviewed README.rst

### PR DESCRIPTION
**Pull request by camerons**
Reviewed README.rst and explained that this doc is about building osgeolive docs, added some missing dependencies to steps, there are still some errors in the build process, fixed some formatting, clarified some advice

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,42 +1,42 @@
-OSGeo live documentation
-========================
-OSGeoLive_ is a self-contained bootable DVD, USB thumb drive or Virtual
-Machine based on Lubuntu, that allows you to try a wide variety of open source
-geospatial software without installing anything. It is composed entirely of
-free software, allowing it to be freely distributed, duplicated and passed
-around.
+Building OSGeo live documentation
```
Comment (cvvergara) [inline README.rst]: Has a space

File: README.rst [hunk 2]
```diff
@@ -1,42 +1,42 @@
-OSGeo live documentation
-========================
-OSGeoLive_ is a self-contained bootable DVD, USB thumb drive or Virtual
```
Comment (cvvergara) [inline README.rst]: Why did you remove the description of OSGeoLive from the README?
Its the first thing people will read, it tells people what the repository is about

**Comment by camerons** [comment]
@cvvergara fair point, I've put back the "About OSGeoLive" text and corrected OSGeoLive spelling.

**Comment by cvvergara** [comment]
@camerons 
the way you worked out the change, affect the history of the paragraph that you removed and placed back.
The blame indicates that Johan Van de Wauw was the original person writing that paragraph.
With a commit removing the paragraph, and a commit putting it back, the blame would say its you the author of that paragraph, taking away the credit of the original person that wrote it.

Can you please do the fixes you intend to do on another branch, and with a new PR.

PS you are starting to experience why I make so many "experimental" branches in my fork, I do that until I get things right.
